reckon-gradle/src/main/java/org/ajoberstar/reckon/gradle/ReckonExtension.java

```diff
@@ -15,16 +15,36 @@
  */
 package org.ajoberstar.reckon.gradle;
 
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import org.ajoberstar.grgit.Grgit;
 import org.ajoberstar.reckon.core.NormalStrategy;
 import org.ajoberstar.reckon.core.PreReleaseStrategy;
 import org.ajoberstar.reckon.core.Reckoner;
 import org.ajoberstar.reckon.core.VcsInventorySupplier;
+import org.ajoberstar.reckon.core.git.GitInventorySupplier;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.ajoberstar.reckon.core.Scope'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AUselessImportCheck)